docs/CPS/CPS-XXXX/src/lace-sequence-flow.puml

```diff
@@ -0,0 +1,54 @@
+@startuml Lace Fundamental State Sequence Flow
+
+actor Lace
+participant "Provider Server" as server
+participant "Cardano Node" as Node
+participant "Cardano DB Sync" as dbSync
+database PostgreSQL as DB
+participant "Token Metadata Service" as tokenMetadataService
+
+group ChainSync
+  group startup
+    dbSync -> DB: queryLastBlock()
+    DB -> dbSync: lastBlock
+    dbSync -> dbSync: convertToPoint(lastBlock)
+    dbSync -> Node: findIntersection(point)
+    Node -> dbSync: intersection
+  end
+  dbSync -> Node: nextBlock(point)
+  Node -> dbSync: block
+  dbSync -> dbSync: decode and process
+  dbSync -> DB: write()
+end
+
+group Long Poll
```

Comment:
Lace is not actually "long polling": that is a technique where a request is kept open until new data is available, and we're not doing that.
```suggestion
group Poll
```

Reply:
https://github.com/will-break-it/wallet-architecture/pull/3/commits/c943e808354db4e5f9dcd714f3fadefbb6979f00